cmd/relayfile-cli/main.go

```
@@ -1157,32 +1148,61 @@ func ensureCloudCredentials(cloudAPIURL, explicitToken string, timeout time.Dura
+		hint = fmt.Sprintf("%s is set to %q — point it at the agent-relay CLI, not the relay broker (agent-relay-broker), or unset it to use PATH.", agentRelayCLIOverrideEnv, bin)
+	}
+	return fmt.Errorf(
+		"agent-relay CLI probe failed: ran `%s` using %q (resolved from %s) and it failed with: %s. relayfile needs `%s` for workspace resolution. %s",
+		probed, bin, origin, detail, probed, hint,
+	)
 }
 
 func ensureAgentRelayCLICompatible() error {
-	bin := agentRelayBinary()
+	bin, origin := agentRelayBinary()
 	versionOutput, err := exec.Command(bin, "--version").CombinedOutput()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)